parsec/core/backend_connection/transport.py

```diff
@@ -114,7 +127,280 @@ async def _connect(
     return transport
 
 
-def _upgrade_stream_to_ssl(raw_stream, hostname):
+def cook_basic_auth_header(username: str, password: str) -> str:
+    step1 = f"{username}:{password}".encode("utf8")
+    step2 = b64encode(step1).decode("ascii")
+    return f"Basic {step2}"
+
+
+# Global vars to easily set up proxy from config file or cli params
```

Comment:
Is this a quick hack? It feels like such parameters are typically managed in the configuration and set through CLI params.

Reply:
removed, this was needed to provide pramas in the cli, but we can work with env var only instead